wagtailio/static/sass/components/_text-media.scss

```diff
@@ -0,0 +1,72 @@
+.text-media {
+    $root: &;
+
+    &__content {
+        margin: auto;
+        grid-row: 2;
+        grid-column: 2 /span 2;
+        padding: 10px 10px 0 0;
+
+        @include media-query(medium) {
+            grid-column: 2/ span 2;
+            grid-row: 1;
+            padding: 0 40px 0 0;
+        }
+
+        @include media-query(large) {
+            grid-column: 2 / span 3;
+            grid-row: 1;
```

Comment:
suggestion: remove this as it's the set on line 12

Reply:
A fair point!